src/analysis/retail/monk/mistweaver/modules/spells/UpliftedSpirits.tsx

```diff
@@ -121,7 +140,7 @@ class UpliftedSpirits extends Analyzer {
         }
       >
         <BoringSpellValueText spell={TALENTS_MONK.UPLIFTED_SPIRITS_TALENT}>
-          <ItemHealingDone amount={healing} />
+          <ItemHealingDone amount={healing + this.effectiveHealingIncrease} />
```

Comment:
I understand the logic but this might confuse people, it might be worth splitting up in the HPS of the statistic itself